sms/v1/openapi.yaml

```diff
@@ -187,8 +185,8 @@ paths:
           in: query
           description: Validity in minutes for this message
           schema:
-            type: string
-            default: "2880"
+            type: number
+            default: 2880
```

Comment:
Could not test it. Is this correct?

Reply:
Your correction is valid, should be number